pkg/resource/events-logger-config/alloy/events-logger.alloy.template

```diff
@@ -37,13 +37,13 @@ loki.process "default" {
 // Loki target configuration
 loki.write "default" {
 	endpoint {
-		url                = nonsensitive(remote.kubernetes.secret.credentials.data["{{ .LoggingURLKey }}"])
+		url                = convert.nonsensitive(remote.kubernetes.secret.credentials.data["{{ .LoggingURLKey }}"])
```

Comment:
This is not supported by old alloy versions so either we need to upgrade the alloy image with the operator or we need to feature flag this use of convert

Reply:
Oh because some cluster are again on old Alloy version ?